src/database/mod.rs

```diff
@@ -494,7 +491,10 @@ pub mod test {
         let (chain, child) = db.del_path_from_script_pubkey(&script).unwrap().unwrap();
 
         assert_eq!(chain, keychain);
-        assert_eq!(path, child);
+        assert_eq!(child, path);
+
+        let res = db.get_path_from_script_pubkey(&script).unwrap();
+        assert!(res.is_none());
```

Comment:
Same for this test additions (and more below): should be in https://github.com/bitcoindevkit/bdk/commit/fd48469b617b2a0335d8a982d913434e8b6df77e, not https://github.com/bitcoindevkit/bdk/commit/2ea1f574928b85ee00829efa9cfc4b23e8a712df